_posts/2019-02-20-bitcoin-abc-0.19.0.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: Prepare for the protocol upgrade today with 0.19.0
+subtitle: May 15th 2019 protocol upgrade is now locked in
+multiLangId: 2019-02-20-bitcoin-abc-0-19-0
+lang: en
+---
+
+The goal of Bitcoin ABC is to help create sound money that is usable globally in order to drive freedom and prosperity for everyone. This [vision](/2018-08-24-bitcoin-abc-vision) has not changed.
+
+Keeping in line with the [Bitcoin Cash roadmap](https://www.bitcoincash.org/roadmap.html), the Bitcoin ABC team is proud to present the May 2019 protocol upgrade release. Bitcoin ABC 0.19.0 is now available and can be downloaded at: [https://download.bitcoinabc.org/0.19.0](https://download.bitcoinabc.org/0.19.0/)
+
+The features going into every protocol upgrade are planned, specified [1], and developed far in advance in order to give users, businesses, miners, and all other ecosystem participants ample time to prepare. In order to assist technical preparations for the upgrade, a testnet is available where the upgrade features can be trialed and tested. Businesses and services built on the Bitcoin Cash network can use this testnet to ensure compatibility, or to get started building products to make use of the upgrade features. Testnet information is available [here](https://github.com/bitcoincashorg/bitcoincash.org/blob/master/workgroups/wg-testing/2019-05-15_upgrade_testnet.md).
```

Comment:
I'd add the link to the specification directly here

Reply:
Good ideas about the direct links